blop/_version.py

```diff
@@ -13,5 +13,5 @@
 __version_tuple__: VERSION_TUPLE
 version_tuple: VERSION_TUPLE
 
-__version__ = version = "0.6.1.dev0+ga388326.d20240131"
-__version_tuple__ = version_tuple = (0, 6, 1, "dev0", "ga388326.d20240131")
+__version__ = version = "0.5.1.dev49+g3f61750.d20240131"
+__version_tuple__ = version_tuple = (0, 5, 1, "dev49", "g3f61750.d20240131")
```

Comment:
Is it updated automatically? I don't think we want such updates in the version history.